WordPress/src/main/res/values/strings.xml

```diff
@@ -2906,6 +2906,8 @@
     <string name="dialog_edit_story_unavailable_message">Unable to load media for this story. Check your internet connection and try again in a moment.</string>
     <string name="dialog_edit_story_unrecoverable_title">Can\'t edit Story</string>
     <string name="dialog_edit_story_unrecoverable_message">We couldn\'t find the media for this story on the site.</string>
+    <string name="dialog_edit_story_unsupported_format_title">Unsupported format</string>
```

Comment:
```suggestion
    <string name="dialog_edit_story_unsupported_format_title">GIF files not supported</string>
```
I was thinking that we could say GIF files here instead of unsupported format. Saying GIF in the title tells the user what went wrong as they see the dialog.